tests/constraint_tests.py

```diff
@@ -178,6 +178,31 @@ def test_linear_transformer_invest(self):
 
         self.compare_lp_files("linear_transformer_invest.lp")
 
+    def test_nonconvex_invest_transformer(self):
+        """Non-convex invest flow with offset, without minimum."""
+        bfuel = solph.buses.Bus(label="fuelBus")
+        bel = solph.buses.Bus(label="electricityBus")
+
+        solph.components.Transformer(
+            label="transformer_nonconvex_invest",
+            inputs={bfuel: solph.flows.Flow()},
+            outputs={
+                bel: solph.flows.Flow(
```

Comment:
Shouldn't it be `NonConvexInvestmentFlow` ?
```suggestion
                bel: solph.flows.NonConvexInvestmentFlow(
```

Reply:
Maybe I will retract this comment as I advance in the review (I am making it now because I don't know if I would have time to finish reviewing all files tonight)